site/kube-addon.md

```diff
@@ -51,11 +58,25 @@ Shut down Kubernetes, and _on all nodes_ perform the following:
 Then relaunch Kubernetes and install the addon as described
 above.
 
-The URL [https://git.io/weave-kube](https://git.io/weave-kube) points
+The URLs https://git.io/weave-kube and https://git.io/weave-kube-1.6 point
 to the YAML file for the latest release of the Weave Net addon.
 Historic versions are archived on our [GitHub release
 page](https://github.com/weaveworks/weave/releases).
 
+## Upgrading Kubernetes to version 1.6
+
+In version 1.6, Kubernetes has increased security, so we need to
+create a special service account to run Weave Net.  This is done in
+the file `weave-daemonset-k8s-1.6.yaml` attached to the release.
```

Comment:
Suggestion:
```
attached to the [Weave Net release](https://github.com/weaveworks/weave/releases/latest).
```
which is more specific, as we talk about both Kubernetes and Weave Net in this paragraph.